content/en/docs/Configuration/p8s-jaeger-grafana/jaeger.md

```diff
@@ -52,3 +52,64 @@ If your Jaeger setup differs significantly from the sample add-ons, make sure
 that Istio is also properly configured to push traces to the right URL.
 {{% /alert %}}
 
+## Use Jaeger frontend with Grafana Tempo tracing backend
+
+It is possible to use the Grafana Tempo tracing backend exposing the Jaeger API.
+[tempo-query](https://github.com/grafana/tempo/tree/main/cmd/tempo-query) is a
+Jaeger storage plugin. It accepts the full Jaeger query API and translates these
+requests into Tempo queries.
+
+Since Tempo is not yet part of the built-in addons that are part of Istio, you
+need to manage your Tempo instance.
+
+### Tempo Operator
+
+The [Tempo Operator for Kubernetes](https://github.com/grafana/tempo-operator)
+provides a native Kubernetes solution to deploy Tempo easily in your system.
+
+After installing the Tempo Operator in your cluster, you can create a new
+Tempo instance with the following CRD:
```

Comment:
This is not a CRD (a CustomResourceDefinition).. this is an instance of the tempo CRD - aka a "Custom Resource" or CR:

```suggestion
Tempo instance with the following CR:
```

Reply:
Ooops. I typed too fast. Thanks!